README.md

```diff
@@ -23,6 +23,87 @@ gem install optimizely-sdk
 To access the Feature Management configuration in the Optimizely dashboard, please contact your Optimizely account executive.
 
 ### Using the SDK
+
+To use the SDK, the Optimizely instance can be initialized in three different ways as per your requirement. Ruby SDK also includes a factory class, OptimizelyFactory, which provides methods to create Optimizely instance with default configuration.
+
+ 1. Initialize Optimizely with a datafile. This datafile will be used as ProjectConfig throughout the life of Optimizely instance.
+    ~~~~~~~~~~~~
+    optimizely_instance = Optimizely::Project.new(datafile)
+2. Initialize Optimizely by providing an 'sdk_key'. This will initialize an HTTPConfigManager that makes an HTTP GET request to the URL ( formed using your provided sdk key and the default datafile CDN url template) to asynchronously download the project datafile at regular intervals and update ProjectConfig when a new datafile is recieved. 
```

Comment:
Do you need a blank line before the second method?

Also, there is an extra space inside the open-parenthesis.

Should "sdk key" be in tick marks?
...using your provided `sdk_key`